addons/point_of_sale/static/src/js/models.js

```diff
@@ -909,6 +910,21 @@ exports.PosModel = Backbone.Model.extend({
         });
         productModel.loaded(this, products);
     },
+
+    // load the partners based on the ids
+    async _loadPartners(partnerIds) {
+        var fields = _.find(this.models, function(model){ return model.label === 'load_partners'; }).fields;
```

Comment:
To get rid of the runbot error, perhaps you should only make rpc if length of partnerIds is greater than or equal to 1.